docs/interfaces/auth-service.txt

```diff
@@ -0,0 +1,45 @@
+Interface Token {
+    payload: {
+        // The lifetime of the Token in milliseconds.
+        expiresIn: number (ms),
+        // The corresponding userId of the requesting client.
+        userId: string,
+        // The id of the current session.
+        sessionId: string
+    },
+    signature: string
+}
+
+/**
+* The client can login with its credentials for authentication.
+* If they are correct, the service answers with a signed Token and sets a cookie, containing the sessionId of the client. 
+*/
+(Token, Cookie) Login (username: string, password: string);
```

Comment:
```suggestion
Login(username: string, password: string): (Token, Cookie);
```
Please use this formatting

Also `login` may raise an error, if the login wasn't successful.